File: PSET 1/mario.c/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int n;
    //GETTING JUST NUMBERS BETWEEN 1 AND 8.
    do
    {
        n = get_int("Height: ");
    }
    while (n < 1 || n > 8);
    // FOR TO RUN LINES
    for (int i = 0; i < n; i++)
    {
        //FOR TO RUN COLUMNS
        for (int j = 0; j < n + i + 3; j++)
        {
            if (j < n - i - 1 || j == n || j == n + 1) 
            {
                //PRINTING SPACE
                printf(" ");
            }
            else
            {
                //PRINTING HASHES
                printf("#");
            }
        }
        //\n TO GO NEXTLINE.
        printf("\n");
    }
}

File: PSET 6/mario.py/mario.py
from cs50 import get_int

height = get_int("Height: ")

# Get just numbers between 1 and 8
while height < 1 or height > 8:
    height = get_int("Height: ")

#Printing
for i in range(1,height + 1):
    print(" " * (height - i), end='')
    print("#" * (i))